### ISSUE TITLE
Test NextCloud over exFAT

### ISSUE THREAD

**Issue by rgaudin** [offspot/base-image#5]
Following #3, we need to secure that NextCloud works fine on exFAT

**Comment by fheslouin** [comment]
Nextcloud require it working directory to be set to permission `0770`, ExFAT does not handle permissions.

The current error message is

```
Error
    Your data directory is readable by other users.
    Please change the permissions to 0770 so that the directory cannot be listed by other users.
```

We can workaround this error by modifying : `'check_data_directory_permissions' => false,` in `config.php`, this is not convenient and not a good practice too, but we can still do it if we re-package nextcloud for our devices. 

@letompouce what do you think ?

**Comment by letompouce** [comment]
Moodle and NextCloud both proved we need a permissions/ownership enabled filesystem, which is addressed in #34 .
